### ISSUE TITLE
Bug still exists in latest .NET 8 Preview 4

### ISSUE THREAD

**Issue by chassq** [dotnet/aspnetcore#48712]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

This bug still exists in .net 8 preview 4: https://github.com/dotnet/aspnetcore/issues/39766

### Expected Behavior

See previous post but when passing HttpContext as the only parameter the api call does not make it into swagger. Adding any other param causes the method to show.

--SHOWS IN SWAGGER
            app.MapGet($"/myapi", async (HttpResponse response, HttpContext context) =>
            {
                var s = "";
            })

--DOES NOT SHOW IN SWAGGER
            app.MapGet($"/myapi", async (HttpContext context) =>
            {
                var s = "";
            })

### Steps To Reproduce

See above.

### .NET Version

8.0.0-preview.4.23260.4

**Comment by captainsafia** [comment]
@chassq Yes, this bug is unfortunately still around. It's being tracked at https://github.com/dotnet/aspnetcore/issues/44970.
